setup.py

```diff
@@ -40,7 +40,7 @@ def utility_wrappers():
             "select_test.x",
             "summary.x",
         )
-    ]
+    ] + ["log-file = ecl.log_util_abort:main"]
```

Comment:
Please don't add a Equinor-specific executable to the pip package. We are exposing too much as it is. I'd recommend going the `set_abort_handler` route where you'd specify a Python function to call and feed directly to the logger instead of this dance where:

1. You add a new separate publicly facing executable script
2. You add a plugin system for a single-use hook

Would prefer to have: `ecl.set_abort_handler` that does just that, and can be set by ERT's azure logging.